src/game/game.cpp

```diff
@@ -86,6 +87,8 @@ Game Game::simulate(
     }
     coins_left -= price;
     attackers.push_back(Attacker::construct(attacker_type, position));
+    Logger::log_spawn(attackers.back().get_id(), attacker_type,
```

Comment:
Moving this log_spawn to Attacker::construct would be better I feel. Lets not have any logger calls from the Game::simulate function at all